java/runtime/src/main/java/io/ray/runtime/functionmanager/FunctionManager.java

```diff
@@ -34,7 +33,7 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-/** Manages functions by job id. */
+/** Manages functions globally. */
```

Comment:
```suggestion
/** Manages functions in the current worker. */
```
globally sounds ambiguous.